.github/workflows/ci.yml

```diff
@@ -7,34 +7,64 @@ on:
     tags:
         - v*
     paths-ignore:
-      - 'README.md'
-      - '.gitlab-ci.yml'
+      - '**.gitignore'
+      - '.images/**'
+      - '**README.md'
+
   pull_request:
     branches:
       - main
+    paths-ignore:
+      - '**.gitignore'
+      - '.images/**'
+      - '**README.md'
+
 
 jobs:
   check:
     name: examples
     strategy:
       matrix:
         platform:
-          - ubuntu_20_64_core
+          - ubuntu-latest
           - macos-latest
         toolchain:
           - stable
-        job:
-          - contract build
-          - test
+        contract:
+          - basic-contract-caller
+          - call-runtime
+          - conditional-compilation
+          - contract-terminate
+          - custom-allocator
+          - custom-environment
+          - dns
+          - e2e-call-runtime
+          - erc20
+          - erc721
+          - erc1155
+          - flipper
+          - incrementer
+          - multi-contract-caller
+          - multi-contract-caller/accumulator
+          - multi-contract-caller/adder
+          - multi-contract-caller/subber
+          - multisig
+          - payment-channel
+          - psp22-extension
+          - rand-extension
+          - trait-dyn-cross-contract-calls
+          - trait-erc20
+          - trait-flipper
+          - trait-incrementer
+          - upgradeable-contracts/delegator
+          - upgradeable-contracts/set-code-hash
+          - vesting
```

Comment:
The only annoying thing here is that we have to remember to update this list when adding/removing examples.

In `ink` I have [this script](https://github.com/paritytech/ink/blob/master/scripts/for_all_contracts_exec.sh) to solve that, although wouldn't be usable in this case because it may not work on OSX and obviously not on Windows.